F1WM/ApiModel/News/NewsType.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace F1WM.ApiModel
+{
+    public class NewsType
+    {
+        public uint Id { get; set; }
+        public string Title { get; set; }
+        public string Title2 { get; set; }
```

Comment:
It causes AutoMapperMappingException at the moment. Anyway, it's redundant, isn't it?